web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -719,6 +719,12 @@ export const formatters: Formatters = {
     format: ({ user, name }) =>
       `User [${user}] has created a trusted relationship with cluster [${name}]`,
   },
+  [eventCodes.PROVISION_TOKEN_CREATED]: {
+    type: 'join_token.create',
+    desc: 'Join Token Created',
+    format: ({ user, roles }) =>
+      `User [${user}] has created a join token with role(s) [${roles}]`,
```

Comment:
Is `${roles}` going to render in user-friendly format or developer language?

Should we do `roles,join(', ')` instead?

Reply:
It's joined by commas by default anyway.
![Screenshot 2023-07-07 at 09 34 22](https://github.com/gravitational/teleport/assets/9391503/7d34b919-351f-4d8c-9321-fd5d4a253dd8)